src/components/course/course-header/CourseHeader.jsx

```diff
@@ -47,6 +48,7 @@ const CourseHeader = () => {
   const isCourseAssigned = useIsCourseAssigned();
   const isCourseArchived = courseMetadata.courseRuns.every((courseRun) => isArchived(courseRun));
   const [partners] = useCoursePartners(courseMetadata);
+  usePassLearnerCsodParams(enterpriseCustomer.uuid);
```

Comment:
[clarification/suggestion] Just curious about the decision to make the API call within the `CourseHeader` component, when this doesn't seem specific to the course header itself? Might it make more conceptual sense to render this hook within the `CourseAbout` component since that's the entry point in the component tree for the course about page?

Reply:
If this holds true it would be great if we could get a quick follow on Ejaz. Thanks!